rca/utils/blocks.py

```diff
@@ -18,6 +18,33 @@ class FeeBlock(blocks.StructBlock):
     )
 
 
+class SlideBlock(blocks.StructBlock):
+    image = ImageChooserBlock()
+    title = blocks.CharBlock(required=False)
+    type = blocks.CharBlock(required=False)
+    summary = blocks.TextBlock(required=False)
+
+    class Meta:
+        icon = "plus"
+
+
+class StatisticBlock(blocks.StructBlock):
+
+    summary = blocks.CharBlock(
+        required=False,
+        help_text="E.g,  1 in 3 of our graduates are business owners or independent professionals",
+    )
+    before = blocks.CharBlock(required=False)
+    after = blocks.CharBlock(required=False, help_text="E.g, '%'", max_length=2)
+    number = blocks.CharBlock(required=False, help_text="E.g, '33'", max_length=5)
+    meta = blocks.CharBlock(
+        required=False, help_text="Small title below the number E.g, 'Nationalities'"
```

Comment:
`Small title below the number E.g, 'Nationalities'` ->  `Small title below the number, e.g 'Nationalities'`

No comma after `e.g.` - that's not British English. No point capitalising `e.g.` if it's not at the beginning of the sentence. I'd add a comma before `e.g.`, if it's part of that sentence.

https://en.wiktionary.org/wiki/e.g.